neo4j/config/driver.go

```diff
@@ -50,7 +51,33 @@ type Config struct {
 	// This is considered an advanced setting, use it at your own risk.
 	// Introduced in 5.0.
 	TlsConfig *tls.Config
-
+	// ClientCertificateProvider specifies a provider for the client TLS certificate for mutual TLS authentication.
+	// Use auth.NewStaticClientCertificateProvider for static certificates, or auth.NewRotatingClientCertificateProvider
+	// for certificates that require runtime updates without application restart.
+	//
+	// Static certificate example:
+	//   config.ClientCertificateProvider = auth.NewStaticClientCertificateProvider(auth.ClientCertificateProvider{
+	//       CertFile: "path/to/client.crt",
+	//       KeyFile:  "path/to/client.key",
+	//   })
+	//
+	// Rotatable certificate example:
+	//   provider, _ := auth.NewRotatingClientCertificateProvider(auth.ClientCertificateProvider{
+	//       CertFile: "path/to/initial_client.crt",
+	//       KeyFile:  "path/to/initial_client.key",
+	//   })
+	//   config.ClientCertificateProvider = provider
+	//   // Dynamically update the certificate later without restarting the application.
+	//   provider.UpdateCertificate(auth.ClientCertificateProvider{
```

Comment:
```suggestion
	//   _ = provider.UpdateCertificate(auth.ClientCertificateProvider{
```
Or possibly `:=`?

Reply:
assignment only is right for the example 👍